### PR TITLE
Fix floating point error in display for credit transfer devices

### LINKED ISSUE #21370: Credit Transfer Devices do not format floats on tip percentages
### Labels

[BUG][MINOR]

### Description

7.000000000001 Appears when a Tip of 7%

### Steps to reproduce

1. Set Amount to 1500
2. Set Tip to 7%

### Expected Behavior

I'd expect a formated value of 7%

### Additional Information & Screenshots

Reported by: marksman2539
Client version: 515.1646
On server: Goonstation 3 Roleplay: Morty
Active test merges: #21006
Round log date: 2024-11-19-10-22
Reported on: 2024-11-19 10:48:06
Round ID: 193359
Map: COGMAP

### PR DESCRIPTION
**Pull request by MyNameIsRomayne**
## About the PR 

Uses toFixed() to round the displayed tip value to the nearest whole number. Only whole number tips can be entered, so this shouldn't have precision issues with that.

Before:
387619958-0acbe955-f02a-4a06-9dd5-ccb9778001c2

After:
image

## Why's this needed? 

Fixes #21370

### CODE REVIEW

File: code/obj/machinery/manufacturer.dm
Comment (pgmzeta) [inline code/obj/machinery/manufacturer.dm:1]: unrelated change bundled in?

Comment (MyNameIsRomayne) [reply code/obj/machinery/manufacturer.dm:1]: Yeah forgot to change branch lol. fixed

**Review by pgmzeta — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]
